compiler/optimizer/LocalAnalysis.cpp

```diff
@@ -714,7 +714,12 @@ bool TR_LocalAnalysisInfo::isCallLike(TR::Node *node) {
 
    if (node->getOpCode().hasSymbolReference())
       {
-      if (node->getSymbolReference()->getSymbol()->isVolatile() || node->getSymbolReference()->getSymbol()->isMethodMetaData())
+      if (node->getSymbolReference()->getSymbol()->isVolatile() || 
+          (node->getSymbolReference()->getSymbol()->isMethodMetaData()
+#if J9_PROJECT_SPECIFIC
```

Comment:
ImmutableField flag is now added into OMRSymbol.hpp and allows it to be used on not only MetaDataSymbol but other symbols as well.  Currently, set this flag only for currentThread symbol.